client/webserver/site/src/js/markets.ts

```diff
@@ -187,6 +197,7 @@ export default class MarketsPage extends BasePage {
     // that the screen is updated with the most recent one.
     this.maxOrderUpdateCounter = 0
     this.metaOrders = {}
+    this.recentMatches = []
```

Comment:
should this be stored at the webserver so we can keep this information in case of refresh?